### PR TITLE
Add more information about security issues

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: contributing/code/security.rst [hunk 1]
```diff
@@ -13,6 +13,28 @@ bug tracker and don't publish it publicly. Instead, all security issues must
...
+  enumeration, login throttling bypasses, denial of service attacks, or timing
+  attacks.
+
+In any case, the core team has the final decision on which issues are
+considered security vulnerabilities.
+
+Security Bug Bounties
+---------------------
+
+Symfony is an Open-Source where most of the work is done by volunteers. We
```
Comment (wouterj) [inline contributing/code/security.rst]: "an Open-Source **project**" ?

**Review by wouterj — APPROVED** [review]
Thank you!

File: contributing/code/security.rst [hunk 2]
```diff
@@ -13,6 +13,28 @@ bug tracker and don't publish it publicly. Instead, all security issues must
 be sent to **security [at] symfony.com**. Emails sent to this address are
 forwarded to the Symfony core team private mailing-list.
 
+The following issues are not considered security issues and should be handled
+as regular bug fixes (if you have any doubts, don't hesitate to send us an
+email for confirmation):
+
+* Any security issues found in debug tools that must never be enabled in
+  production (including the web profiler or anything enabled when `APP_DEBUG`
```
Comment (javiereguiluz) [inline contributing/code/security.rst]: Please, double backticks for the code elements:

```
``APP_DEBUG``
```

Thanks!

Comment (fabpot) [reply contributing/code/security.rst]: Oops, done

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks Fabien.

**Pull request merged**